packages/console/src/widget.ts

```diff
@@ -846,6 +879,20 @@ export class CodeConsole extends Widget {
     }
   }
 
+  /**
+   * Update the console node with class indicating read-write state.
+   */
+  private _updateReadWrite(): void {
+    // TODO: de-duplicate with code in console/src/widget.ts
+    const inReadWrite = DOMUtils.hasActiveEditableElement(this.node);
+    const hasReadWriteClass = this.node.classList.contains(READ_WRITE_CLASS);
+    if (inReadWrite && !hasReadWriteClass) {
+      this.node.classList.add(READ_WRITE_CLASS);
+    } else if (!inReadWrite && hasReadWriteClass) {
+      this.node.classList.remove(READ_WRITE_CLASS);
```

Comment:
Why can't you do 

```suggestion
    this.node.classList.toggle(READ_WRITE_CLASS, inReadWrite);
```

like you do in the notebook code?